pcdsdevices/beam_stats.py

```diff
@@ -75,3 +84,98 @@ def __init__(self, prefix, *, name, skip_small_moves=True, atol=None,
             self.atol = atol
         super().__init__(prefix, name=name, skip_small_moves=skip_small_moves,
                          **kwargs)
+
+
+class Lcls(BaseInterface, Device):
+    '''
+    LCLS LINAC STATUS
+    bunch charge: 180 [pC]
+    repetition rate: 120 [Hz]
+    ebeam energy: 10.75 [GeV]
+    Vernier energy: -35 [MeV]
+    Photon Energy: 9.91 [keV]
+    BC2 peak current: 3201 [A]
+    electron bunch length: 23 [fs]
+    Last eLoss Scan: 0.93 [mJ]
+    '''
+    # Bunch Charge nC: FBCK:BCI0:1:CHRG	no desc
+    # Hard e-Energy GeV/c: BEND:DMPH:400:BDES - Desired B-Field
+    # Soft e-Energ GeV/c: BEND:DMPS:400:BDES - Desired B-Field
+    # Bunch Length fs: SIOC:SYS0:ML00:AO820 estimated FEL Pulse Duration (FWHM)
+
+    # BC2 IPK:	 FBCK:FB04:LG01:S5DES	BC2 Current set point
+
+    # Bunch charge (units: nC)
+    bunch_charge = Cpt(EpicsSignalRO, 'SIOC:SYS0:ML00:AO470', kind='normal')
```

Comment:
Whitelist components and methods for tab completion:
```suggestion
    tab_component_names = True
    tab_whitelist = ['... add your method names from below ...']
    bunch_charge = Cpt(EpicsSignalRO, 'SIOC:SYS0:ML00:AO470', kind='normal')
```

Reply:
Cool, I added all the methods here in e13f86b, thank you.